config/betterquesting/DefaultQuests.json

```diff
@@ -14572,7 +14410,7 @@
           "ismain:1": 1,
           "repeat_relative:1": 1,
           "name:8": "Refined Processors",
-          "desc:8": "This is the final LV Circuit."
+          "desc:8": "This is the final LV Circuit, and the beginning the of \"Refined\" circuit tier.\n\nFrom now on, all the circuits tiers will follow an identical pattern.\n\nLevel 1 - Processor\nLevel 2 - Assembly\nLevel 3 - Computer\nLevel 4 - Mainframe"
```

Comment:
The second sentence's "circuits" should have an apostrophe at the end. Note that you cannot put an apostrophe directly in the json file, so you need to use `\u0027ll` instead.

Reply:
Also changed "the of" to "of the", so many typos :weary: